### ISSUE TITLE
Title of popups is always "undefined" in 1.2.0

### ISSUE THREAD

**Issue by jncanches** [thomasloven/hass-browser_mod#154]
My Home Assistant version: 0.116.4

What I am doing:
Call service browser_mod.popup with a title set

What I expected to happen:
Popup title displayed as expected

What happened instead:
Popup displayed "undefined" as title, no matter what is set in yaml config

```yaml
    action: call-service
    service: browser_mod.popup
    service_data:
      deviceID: this
      title: My title
      card:
        whatever you want
```

Error messages from the browser console:
No error message in chrome Dev-Tools console

---

**By putting an X in the boxes ([ ]) below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by mikepan-ovoenergy** [comment]
Confirm!

**Comment by pollinolas** [comment]
same here,thanks to open de issue

**Comment by ASNNetworks** [comment]
Also global popups (replace the more-info) don’t work at all. So nothing happens when invoking the global popup. Downgrading solves it for now.

**Comment by jumping2000** [comment]
same here!

**Comment by ASNNetworks** [comment]
1.2.1 doesn’t fix this unfortunately.

**Comment by gurbyz** [comment]
🥳 For me version 1.2.1 fixed it in my (very old dashboard) browser version `Chromium 78.0.3904.108 Built on Raspbian , running on Raspbian 10` immediately (well, I did a restart and a reload resources).

On my Mac I had to upgrade Chrome to `Version 86.0.4240.111` and then the popup titles appeared again. I don't know on which version a was before upgrading, but I restarted, reloaded resources, refreshed using the HA-menu en refreshed the browser. That didn't help. Didn't clean cache, but upgrading Chrome did help.
